src/Samples/SarifToCsv/Program.cs

```diff
@@ -39,6 +39,8 @@ public static void Main(string[] args)
                 string sarifFilePath = args[0];
                 string csvFilePath = args[1];
                 IEnumerable<string> columnNames = (args.Length > 2 ? args[2] : ConfigurationManager.AppSettings["ColumnNames"]).Split(',').Select((value) => value.Trim());
+                bool removeNewlines = bool.Parse(ConfigurationManager.AppSettings["RemoveNewlines"] ?? "false");
```

Comment:
>?? [](start = 100, length = 2)

Bug: I just tested this and if a setting is missing, `AppSettings["MissingSetting"]` returns the empty string, not `null`.

Reply:
![image](https://user-images.githubusercontent.com/15742033/65177275-40e8ba00-da0b-11e9-857a-aa8bc5db7a8e.png)

How did you get that to happen? In the debugger, I see null coming through and defaulting to false if I remove the setting from the app.config completely.

Are you running on the .NET Core runtime and I'm running .NET Framework?

Reply:
I made a little test app. It runs on .NET Framework 4.6.1. I'll zip it up and mail it to you.

---
In reply to: [325840398](https://github.com/microsoft/sarif-sdk/pull/1672#discussion_r325840398) [](ancestors = 325840398)

Reply:
But I see your change accounts for both possibilities, so that's fine.

---
In reply to: [325909860](https://github.com/microsoft/sarif-sdk/pull/1672#discussion_r325909860) [](ancestors = 325909860,325840398)

Reply:
**UPDATE**: I looked at my test app again and it had a bug. Your original code is correct.

---
In reply to: [325910099](https://github.com/microsoft/sarif-sdk/pull/1672#discussion_r325910099) [](ancestors = 325910099,325909860,325840398)